src/OXRS_LCD.cpp

```diff
@@ -733,3 +759,97 @@ void OXRS_LCD::_set_backlight(int val)
 {
   ledcWrite(BL_PWM_CHANNEL, 255*val/100); 
 }
+
+
+
+
+// Bodmers BMP image rendering function
+
+void OXRS_LCD::_drawBmp(const char *filename, int16_t x, int16_t y) 
+{
+  uint32_t seekOffset;
+  uint16_t w, h, row, col;
+  uint8_t  r, g, b;
+  File bmpFS;
+
+  // Open requested file on SPIFFS
+  bmpFS = SPIFFS.open(filename, "r");
+  if (!bmpFS)
+  {
+    Serial.println(F("[lcd] File not found"));
```

Comment:
I like that you are using the new logging standard! Can you change all instances to use a 4 char (padded) prefix - e.g. `[lcd ] file not found`. And lowercase cos I have crippling OCD... ;)